.github/workflows/PreviewVercelDeploy.yaml

```
@@ -0,0 +1,208 @@
+  VERCEL_ORG_ID: ${{ secrets.VERCEL_ORG_ID }}
+  VERCEL_PROJECT_ID: ${{ secrets.VERCEL_PROJECT_ID }}
+  VERCEL_CLI_TOKEN: ${{ secrets.VERCEL_CLI_TOKEN }}
+  TEST_DATABASE_URL: ${{ secrets.DEVELOPMENT_DATABASE_URL }}
+  NODE_ENV: 'test'
+
+jobs:
+  docker-lint:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)